website/docs/r/dashboard.html.markdown

```diff
@@ -719,6 +730,15 @@ Nested `widget` blocks have the following structure:
         - `title_size`: (Optional) The size of the widget's title. Default is 16.
         - `title_align`: (Optional) The alignment of the widget's title. One of "left", "center", or "right".
         - `time`: (Optional) Nested block describing the timeframe to use when displaying the widget. The structure of this block is described [below](dashboard.html#nested-widget-time-blocks).
+  - `service_level_objective_definition`: The definition for a Service Level Objective widget. Exactly one nested block is allowed with the following structure:
```

Comment:
After updating the fields pointed out by @platinummonkey to be Required, please also update the `(Optional)` => `(Required)` for these.